api/controllers/shabads.js

```diff
@@ -258,6 +258,65 @@ exports.search = async (req, res) => {
   }
 };
 
+exports.resultsInfo = async (req, res) => {
+  const { VerseID } = req.params;
+  const results = 20;
+  let page = parseInt(req.query.page, 10) || 1;
+  if (lib.isListOfNumbers(VerseID)) {
+    let conn;
+    try {
+      conn = await req.app.locals.pool.getConnection();
+
+      const q = `SELECT ${allColumns} ${allFrom}
+           WHERE v.ID IN (${VerseID})`;
```

Comment:
Is it safe from SQL injection? Need to test it.

Reply:
We tested this and there was no issue like that, but we could sanitize the inputs here.

Reply:
Sure.

Reply:
https://github.com/KhalisFoundation/banidb-api/issues/271